### PR TITLE
Usage of wrong Liquid script parameter name

### PR DESCRIPTION
**Pull request by jtkech**
Maybe because it was working with the old tag helper bridge

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Demo/Views/Todo.Edit.liquid [hunk 1]
```diff
@@ -46,5 +46,5 @@
     </table>
 {% endform %}
 
-{% script  src:"https://cdn.jsdelivr.net/npm/jquery-validation@1.19.1/dist/jquery.validate.min.js", at: "Foot", dependsOn: "jquery" %}
-{% script  src:"https://cdn.jsdelivr.net/npm/jquery-validation-unobtrusive@3.2.11/dist/jquery.validate.unobtrusive.min.js", at: "Foot", dependsOn: "jquery" %}
+{% script src:"https://cdn.jsdelivr.net/npm/jquery-validation@1.19.1/dist/jquery.validate.min.js", at: "Foot", depends_on: "jquery" %}
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Demo/Views/Todo.Edit.liquid:49]: `depends_on` or `depends-on`?

Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.Demo/Views/Todo.Edit.liquid:49]: Never use a dash in liquid, you can see all parameter name in the `ScriptTag.cs` file of the OC.Resources project

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Demo/Views/Todo.Edit.liquid:49]: Seem we need to fix them in mny places then

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Demo/Views/Todo.Edit.liquid:49]: Have a look https://github.com/OrchardCMS/OrchardCore/search?q=depends-on

Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.Demo/Views/Todo.Edit.liquid:49]: Yes, we use `depends-on` in razor files when using our custom Script TagHelper

But here we need `depends_on` as we are in a Liquid file using our Liquid ScriptTag

**Comment by sebastienros** [comment]
It's `depends_on` because of this: https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Resources/Liquid/ScriptTag.cs#L52

Maybe check other properties, I might have done the same mistake in other pages.

**Pull request merged**